<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowStability</class>
 <widget class="QMainWindow" name="MainWindowStability">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1079</width>
    <height>815</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string> VirtualRobot - Stability Demo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frameViewer">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>510</y>
         <width>201</width>
         <height>251</height>
        </rect>
       </property>
       <property name="title">
        <string>Visualization</string>
       </property>
       <widget class="QCheckBox" name="checkBoxFootstepPositions">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>141</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Show Footstep Positions</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBoxFeetTrajectories">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>131</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Show Feet Trajectories</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBoxZMPTrajectory">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>131</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Show ZMP Trajectory</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBoxCoMTrajectory">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>120</y>
          <width>131</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Show CoM Trajectory</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBoxZMP">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>150</y>
          <width>131</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Show ZMP</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBoxCoM">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>180</y>
          <width>131</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Show CoM</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBoxSupportPolygon">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>210</y>
          <width>131</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Show Support Polygon</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_1">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>201</width>
         <height>101</height>
        </rect>
       </property>
       <property name="title">
        <string>Robot</string>
       </property>
       <widget class="QPushButton" name="pushButtonReset">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>171</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonLoad">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>171</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Select Robot File</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>120</y>
         <width>201</width>
         <height>191</height>
        </rect>
       </property>
       <property name="title">
        <string>Parameters</string>
       </property>
       <widget class="QLineEdit" name="lineEditStepPeriod">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>20</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>0.8</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditStepLength">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>50</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>0.3</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditDSLength">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>80</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>0.1</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>StepPeriod</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>StepLength</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>DSLength</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>173</x>
          <y>20</y>
          <width>20</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>s</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>100</y>
          <width>20</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>s</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>70</y>
          <width>20</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>s</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>173</x>
          <y>50</y>
          <width>20</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>m</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>173</x>
          <y>80</y>
          <width>20</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>s</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditStepHeight">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>110</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>0.1</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>173</x>
          <y>110</y>
          <width>20</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>m</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>StepHeight</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonCalculate">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>140</y>
          <width>171</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Calculate</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1079</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
